docs/source/upcoming_release_notes/282-enh_filter_default_kwargs.rst

```diff
@@ -0,0 +1,27 @@
+282 enh_filter_default_kwargs
+#############################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- Add `EntryInfo` keyword argument ``include_default_as_kwarg``.  If set to ``False``,
+  any keys that are included in an item's ``kwargs`` that match the default of their
+  corresponding `EntryInfo` will be omitted from the keyword arguments passed to
+  ``device_class`` when instantiating (loading) the item as in ``happi.loader.load_device`` or
+  ``SearchResult.get()``.
+  The default value is True, retaining the original behavior.
```

Comment:
We should either clarify that this needs to be used on the `kwargs` entry info or allow it to have some associated effect when used on the others. As-written this is not entirely clear about how to invoke this behavior.